src/__tests__/core/web_api/p2_api.test.js

```diff
@@ -78,18 +77,6 @@ describe('Auth0APIClient', () => {
         const loginMock = mock.WebAuth.mock.instances[0].login.mock;
         assertCallWithCallback(loginMock, callback);
       });
-      it('should call client.loginWithResourceOwner when redirect is false and sso is false', () => {
```

Comment:
If this premise is no longer valid, negate the test rather than removing it. Helps keep the coverage 👍